src/theory/quantifiers/sygus/cegis.cpp

```diff
@@ -527,8 +527,9 @@ bool Cegis::sampleAddRefinementLemma(const std::vector<Node>& candidates,
         // mark this as a CEGIS point (no longer sampled)
         d_cegis_sample_refine.insert(i);
         std::vector<Node> vars;
+        d_cegis_sampler.getVariables(vars);
```

Comment:
It looks like `vars` is not used. Can we remove it?

Reply:
Yes good catch.